docs/quickstart.md

```diff
@@ -212,7 +212,7 @@ for step in range(1000):
 ## Evaluation
 
 Now that we have trained our neural network we can evaluate it on the test set.
-We will be using the same batch size of 64 and will be evaluating for 1000 of those batches.
+We will be using the same batch size of 64 and will be evaluating 1000 of those batches.
```

Comment:
for makes sense here tho? 1000 is our step count so for kinda indicates that, rather than just some random 1000 batches.

Reply:
Yes, I see now it makes sense.